docs/development.md

```diff
@@ -103,9 +103,8 @@ This instruction set applies for both dev server and production server, with sli
 
 ### As administrator
 
-1. Go to any administrator page, e.g `/web/admin/home`.
-1. On the dev server, log in using any username, but remember to check the `Log in as administrator` check box. You will have the required access.
-1. On the production server, you will be granted the access only if your account has admin permission as defined in `build.properties`.
+1. Go to any administrator page, e.g `/web/admin/home`. You may be prompted to log in.
+   You will be granted access only if your account has admin permission as defined in `build.properties`.
```

Comment:
I don't think it will be useful. `build.properties` have so many things inside of it and we don't see things like "Please add `app.localdatastore.port=8484` so that your Datastore emulator will run at this port".

If the issue is the effect on current, active developers, then it is a one-time thing that should be communicated in another manner.

Reply:
Yeah, no big deal this.